src/dataviews/category-dataview-model.js

```diff
@@ -17,14 +17,16 @@ module.exports = DataviewModelBase.extend({
     {
       type: 'category',
       filterEnabled: false,
+      categories: 6,
```

Comment:
Perhaps `limit` is a more descriptive word for this?

Reply:
Categories is the word used in the server url

Reply:
That doesn't mean we have to use the same word 😄

Reply:
Yes. We use now `limit` for the public API. But this file is for the internal model.

Reply:
I still find `limit` a better name even for the internal model `dataview.get('categories')` is a bit generic, don't you think?